src/fragments/lib/auth/js/social.mdx

```diff
@@ -177,6 +177,11 @@ function App() {
 }
 ```
 
+<Callout warning={true}>
```

Comment:
This is what it looks like: 
![image](https://user-images.githubusercontent.com/43682783/168703974-aacfc750-d280-40af-93fc-2c06d3e68a59.png)